client/src/pages/index.tsx

```diff
@@ -67,6 +68,15 @@ const combineOr = (...checks: any[]) => (course: Course, role: Role, session: Se
   checks.some(check => check(course, role, session));
 
 const routes = [
+  {
+    name: () => (
+      <>
+        <DashboardTwoTone /> Dashboard
+      </>
+    ),
+    getLink: (course: Course) => `/course/student/dashboard?course=${course.alias}`,
+    access: combineAnd(isCourseNotCompleted, isStudent),
+  },
```

Comment:
please remove it. Let's make it accessible only by direct link for now. When we test it and add all required functionality, then we will enable it on the home page

Reply:
@apalchys  
Thanks, I fixed issue